Makefile

```diff
@@ -1,22 +1,61 @@
+ARCH    ?= amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
+
 .PHONY: all test
 
 IMAGE_BUILD_CMD := docker build
 
 VERSION := $(shell git describe --tags --dirty --always)
 
+IMAGEARCH ?=
+QEMUARCH  ?=
+
+ifeq ($(ARCH),amd64)
+        IMAGEARCH =
+        QEMUARCH  = x86_64
+else ifeq ($(ARCH),arm)
+        IMAGEARCH = arm32v7/
+        QEMUARCH  = arm
+else ifeq ($(ARCH),arm64)
+        IMAGEARCH = arm64v8/
+        QEMUARCH  = aarch64
+else ifeq ($(ARCH),ppc64le)
+        IMAGEARCH = ppc64le/
+        QEMUARCH  = ppc64le
+else ifeq ($(ARCH),s390x)
+        IMAGEARCH = s390x/
+        QEMUARCH  = s390x
+else
+	$(error unknown arch "$(ARCH)")
```

Comment:
Change the tab indentation here to spaces

Reply:
Done.